hathor/manager.py

```diff
@@ -191,6 +208,8 @@ def start(self) -> None:
         self.tx_storage.disable_lock()
         # Initialize manager's components.
         self._initialize_components()
+        if self._full_verification:
+            self.tx_storage.disable_full_verification()
```

Comment:
Why do we need to disable it?

Reply:
Before calling `self._initialize_components()` I enter 'full verification mode' and after that I disable it. It's just to know if I've stopped a full initialization in the middle

Reply:
I've changed to `start_full_verification` and `finish_full_verification`. I think the names enable/disable were not the best options, as Jan said

Reply:
I got it. Ok! Can you drop a comment explaining it, please?

Reply:
Done